tests/test_rich_markup_mode.py

```diff
@@ -0,0 +1,40 @@
+import typer
+import typer.completion
+from typer.testing import CliRunner
+
+runner = CliRunner()
+rounded = ["╭", "─", "┬", "╮", "│", "├", "┼", "┤", "╰", "┴", "╯"]
+
+
+def test_rich_markup_mode_none():
```

Comment:
This test currently fails on `master`, which is clearly a bug IMO.

Reply:
yes it fails for 0.12.5 release here as well
```
def test_rich_markup_mode_rich():
        app = typer.Typer(rich_markup_mode="rich")

        @app.command()
        def main(arg: str):
            """Main function"""
            print(f"Hello {arg}")

        assert app.rich_markup_mode == "rich"

        result = runner.invoke(app, ["World"])
        assert "Hello World" in result.stdout

        result = runner.invoke(app, ["--help"])
>       assert any(c in result.stdout for c in rounded)
E       assert False
E        +  where False = any(<generator object test_rich_markup_mode_rich.<locals>.<genexpr> at 0x7f931ae59d80>)

tests/test_rich_markup_mode.py:40: AssertionError
```